packages/test-data/less/import-redirect/import-redirect.less

```diff
@@ -0,0 +1,3 @@
+@import "https://example.com/redirect.less";
```

Comment:
Is this something that will always redirect? As in, will this test always succeed with this remote endpoint?

Reply:
This is the url `nock` mocks, so it doesn't actually send request to a remote server. So as long as `nock` is set up to intercept this exact url it will always redirect.

Reply:
Ah, perfect.